packages/backend/src/tokens/interstitial.ts

```diff
@@ -67,7 +69,7 @@ export function loadInterstitialFromLocal(
                 try {
                     await Clerk.load({
                         isSatellite: ${isSatellite},
-                        shouldSyncLink: ${isSatellite && debugData?.reason !== 'satellite-cookie-missing'},
+                        shouldSyncLink: ${shouldSyncLink},
```

Comment:
🙃 Maybe rename this to 
```suggestion
                        shouldSyncWithPrimary: ${shouldSyncWithPrimary},
```

syncLink sounds a bit weird.